src/olympia/addons/utils.py

```diff
@@ -302,10 +302,13 @@ def webext_version_stats(request, source):
     log.info('webext_version_stats no match')
 
 
-def is_version_number_not_greater_than_current(addon, version_string):
+def validate_version_number_is_greater(addon, version_string):
```

Comment:
I like the `validate_` part, but the fact that it doesn't say `greater` than what is weird...

Reply:
https://github.com/mozilla/addons-server/pull/19971/commits/55d76e55d6dc3ab7ed6aceac7f1dec98ef4538ae `validate_version_number_is_gt_latest_signed_listed_version` is the longest that fits